src/components/nodes-table/SmartView.vue

```diff
@@ -505,10 +505,16 @@ export default {
 				icon = mdiPowerPlug
 				description = 'Main power source'
 			} else {
-				description = Array.isArray(node.batteryLevels)
-					? 'All battery levels: ' +
-					  node.batteryLevels.map((v) => `${v}%`).join(',')
-					: 'Unknown battery level'
+				description =
+					node.batteryLevels !== null &&
+					typeof node.batteryLevels === 'object' &&
+					!Array.isArray(node.batteryLevels)
+						? 'All battery levels: ' +
+						  Object.entries(node.batteryLevels)
```

Comment:
You sort them now but no info about the endpoint?

Reply:
I've changed array to object and key order in object is not defined or predictable - I'm sorting entries by key and combining values as it was previously with array - result should be the same.